packages/indicators/src/__tests__/__integration/arithmeticAnalytics.fixtures.json

```diff
@@ -0,0 +1,447 @@
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2017-01-01",
+          "endDate": "2017-12-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": []
+    },
+    {
+      "description": "Evaluates an arithmetic formula",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2019-01-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "MonthlyBirths",
+          "organisationUnit": "GR",
+          "period": "20190101",
+          "value": 3
+        }
+      ]
+    },
+    {
+      "description": "Evaluates a boolean formula (result: true)",
+      "input": {
+        "indicatorCodes": ["FemaleGtMale"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2019-01-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "FemaleGtMale",
+          "organisationUnit": "GR",
+          "period": "20190101",
+          "value": 0
+        }
+      ]
+    },
+    {
+      "description": "Evaluates a boolean formula (result: false)",
+      "input": {
+        "indicatorCodes": ["MaleGtFemale"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2019-01-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "MaleGtFemale",
+          "organisationUnit": "GR",
+          "period": "20190101",
+          "value": 1
+        }
+      ]
+    },
+    {
+      "description": "Applies correct aggregation (string format)",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2020-01-01",
+          "endDate": "2020-12-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "MonthlyBirths",
+          "organisationUnit": "GR",
+          "period": "20201130",
+          "value": 7
+        },
+        {
+          "dataElement": "MonthlyBirths",
+          "organisationUnit": "GR",
+          "period": "20201230",
+          "value": 14
+        }
+      ]
+    },
+    {
+      "description": "Applies correct aggregation (array format)",
+      "input": {
+        "indicatorCodes": ["AnnualBirths"],
+        "fetchOptions": {
+          "startDate": "2020-01-01",
+          "endDate": "2020-12-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "AnnualBirths",
+          "organisationUnit": "GR",
+          "period": "2020",
+          "value": 21
+        }
+      ]
+    },
+    {
+      "description": "Applies correct aggregation (object format)",
```

Comment:
This test case will fail - and that's a good thing!! Again, I believe we got lucky in PSSS - there are a few bugs related to `fetchOptions` expansion during fetching, and during aggregation. Good news - this is also related to your original comment in the previous PR: https://github.com/beyondessential/tupaia/pull/1873#issuecomment-761892449.

I'll submit a new PR to fix this, and may add a couple of more complicated test cases here (nested indicators, including `OFFSET_PERIOD` and `SUM` aggregations, make sure these work correctly together)

Reply:
Oh ok cool! Nice test case :-)